input/system/selfhosted/logs.go

```diff
@@ -132,13 +132,32 @@ func SetupLogTailForServer(ctx context.Context, wg *sync.WaitGroup, opts state.C
 	return setupLogLocationTail(ctx, server.Config.LogLocation, logStream, logger)
 }
 
-func SetupOtelHandlerForServer(ctx context.Context, wg *sync.WaitGroup, opts state.CollectionOpts, logger *util.Logger, server *state.Server, parsedLogStream chan state.ParsedLogStreamItem) error {
-	if opts.DebugLogs || opts.TestRun {
-		logger.PrintInfo("Setting up OTLP HTTP server receiving logs with %s", server.Config.LogOtelServer)
-	}
+func SetupOtelHandlerForServers(ctx context.Context, wg *sync.WaitGroup, opts state.CollectionOpts, logger *util.Logger, servers []*state.Server, parsedLogStream chan state.ParsedLogStreamItem) {
+	// A list of all OpenTelemetry servers that we have started to prevent
+	var otelServers []string
 
-	logStream := setupLogTransformer(ctx, wg, server, opts, logger, parsedLogStream)
-	return setupOtelHandler(ctx, server, logStream, parsedLogStream, logger)
+	for _, server := range servers {
+		otelLogServer := server.Config.LogOtelServer
+		if otelLogServer == "" {
+			continue
+		}
+
+		prefixedLogger := logger.WithPrefix(server.Config.SectionName)
+
+		if util.SliceContains(otelServers, otelLogServer) {
+			prefixedLogger.PrintInfo("OpenTelemetry log handler on %s already registered, skipping. Check your configuration for duplicate 'db_log_otel_server' entries.", otelLogServer)
+			continue
+		}
+
+		if opts.DebugLogs || opts.TestRun {
+			logger.PrintInfo("Setting up OTLP HTTP server receiving logs with %s", server.Config.LogOtelServer)
```

Comment:
Uhmm, I'm still not so sure about having this line and below line (in otel_hanlder.go) separately.

```
prefixedLogger.PrintVerbose("Registered OpenTelemetry log handler on %s", otelLogServer)
```

How about removing here, then change above line to PrintInfo, and having the condition of printing it out with `opts.DebugLogs || opts.TestRun || opts.VeryVerbose`?
Or, I think it's also okay to simply change the log level to PrintInfo and log all the time too.

Reply:
Fair enough. I honestly didn't notice this preexisting message when we were refactoring the PR and added the verbose message as it seemed like we don't have consistency on what we log at startup - but I like your suggestion to keep it here.